tools/api-generator/pom.xml

```diff
@@ -32,6 +32,11 @@
     </plugins>
   </build>
   <dependencies>
+    <dependency>
+      <groupId>com.microsoft.playwright</groupId>
```

Comment:
This introduces circular dependency as api-generator generates code for playwright but at the same time requires playwright. We should avoid that.

Reply:
OK, I get it. Also that I didn't notice the circular dependency was introduced.
But is there any better way to know about which devices `playwright` can simulate? (except for debug the code)
:) Any reference docs in the upstream there?

Reply:
> But is there any better way to know about which devices `playwright` can simulate? (except for debug the code)
> :) Any reference docs in the upstream there?

https://playwright.dev/docs/api/class-playwright?_highlight=device#playwrightdevices is the upstream documentation. It simply points to the source file with the definitions of the devices. One could always print the list of keys in the map.

Reply:
Thank you, It's clear and simple, now I'm closing this pull request.